### PR TITLE
Do copy/past of task between workflows.

### PR DESCRIPTION
**Pull request by medou-boushab**

### CODE REVIEW

File: app/scripts/proactive/view/WorkflowView.js [hunk 1]
```diff
@@ -509,7 +509,7 @@ define(
                 return (name === view.model.get("Task Name"));
             })
         },
-        copyPasteTasks: function (tasks, position) {
+        copyPasteTasks: function (tasks, position , newTaskModels, Taks) {
```
Comment (MaelAudren) [inline app/scripts/proactive/view/WorkflowView.js]: I don't understand what is `Taks`, maybe you could rename it into something more explicit. I think that you should follow the naming convention and put the variable in lowercase.

File: app/scripts/proactive/view/dom.js [hunk 2]
```diff
@@ -1027,16 +1027,24 @@ define(
...
                         copied.push(t);
+                        task.push($(t).data( "view" ))
                     })
                 }
                 if (ctrlDown && e.keyCode == vKey) {
                     if (pasteAllow) {
-                        console.log("paste");
-                        require('StudioApp').views.workflowView.copyPasteTasks(copied, pasteAllow);
+                        var newTaskModel = []
+                        var Tasksview = [];
```
Comment (MaelAudren) [inline app/scripts/proactive/view/dom.js]: Please follow the naming convention and rename `Tasksview` in `tasksView`

File: app/scripts/proactive/view/dom.js [hunk 3]
```diff
@@ -1027,16 +1027,24 @@ define(
...
+                         $.each(copied, function (i, t) {
```
Comment (chloegugli) [inline app/scripts/proactive/view/dom.js]: As you loop in copied to get elements for task, maybe you don't need copied table anymore. Also, it seems that there are several tasks in 'task' so you might rename it 'tasks'

File: app/scripts/proactive/view/WorkflowView.js (hunk 1 above)
Comment (fviale) [reply app/scripts/proactive/view/WorkflowView.js]: looks like a typo, but yes it should be more understandable. for example tasksToCopy, tasksFromCurrentModel (I don't know the precise semantic here)

File: app/scripts/proactive/view/dom.js [hunk 4]
```diff
@@ -998,7 +998,7 @@ define(
         })();
 
         $(document).ready(function () {
-
+            var copiedTask = [];
```
Comment (chloegugli) [inline app/scripts/proactive/view/dom.js]: There are several tasks inside copiedTask right? 'copiedTasks' might be more appropriate

File: app/scripts/proactive/view/WorkflowView.js (hunk 1 above)
Comment (medou-boushab) [reply app/scripts/proactive/view/WorkflowView.js]: done

**Pull request merged**